backend/slack_commands/execution/session_executor.py

```
@@ -37,15 +37,15 @@ def close(self) -> None:
         user_name: str,
-        blueprint_id: str,
+        workflow_id: str,
         question: str,
         response_url: str,
         *,
         public: bool = False,
     ) -> None:
         """Submit a background task that creates, submits, polls, and responds."""
         self._pool.submit(
-            self._execute, user_name, blueprint_id, question, response_url, False, public,
+            self._execute, user_name, workflow_id, question, response_url, False, public,
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)